core/services/chainlink/config_pytoscope.go

```diff
@@ -0,0 +1,23 @@
+package chainlink
+
+import v2 "github.com/smartcontractkit/chainlink/v2/core/config/v2"
+
+type PyroscopeConfig struct {
+	c v2.Pyroscope
+	s v2.PyroscopeSecrets
+}
+
+func (p PyroscopeConfig) AuthToken() string {
```

Comment:
Can these receivers be pointers? Would save a bit of needless copying.
```suggestion
func (p *PyroscopeConfig) AuthToken() string {
```